lib/ngx/pipe.lua

```diff
@@ -583,7 +576,8 @@ do
 
         return proc
     end
-end  -- do
+end -- do
```

Comment:
Err, I think there should be two spaces after the `end` keyword?

Reply:
Indeed, my bad.